cob_bms_driver/src/fake_bms.py

```diff
@@ -80,7 +80,6 @@ def timer_cb(self, event):
     def timer_consume_power_cb(self, event):
         # emulate the battery usage based on the current values
         self._remaining_capacity += (self._current/self._poll_frequency)/3600.0
-        self._remaining_capacity = round(self._remaining_capacity,3)
```

Comment:
The rounding was introduced in https://github.com/ipa320/cob_driver/commit/2a283127e1b7892731acf6ed4543eb1a5a5f284f (from https://github.com/ipa320/cob_driver/pull/374) and for a good reason I assume. I guess the real driver has lower precision and consumers of the value or representation depends on it. 

So, I'd leave the rounding in for the published data, but do not round the variable itself? Eg. 
```python
self._pub_remaining_capacity.publish(round(self._remaining_capacity, 3))
```
etc. 

Could do this via a `@property` as well like

```
@property
def remaining_capacity(self):
    return round(round(self._remaining_capacity, 3))
```
while still setting unrounded `self._remaining_capacity`

but might also be overkill for the few occurrences that need it.

Reply:
+1

Reply:
That's a good point, I added the rounding for the published data and diagnostics.